arch/arm/boot/dts/overlays/imx378-overlay.dts

```diff
@@ -0,0 +1,21 @@
+// SPDX-License-Identifier: GPL-2.0-only
+// Definitions for IMX378 camera module on VC I2C bus
+/dts-v1/;
+/plugin/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include "imx477_378-overlay.dtsi"
+
+/{
```

Comment:
There's no need to repeat the compatible string. In fact, because the change of compatible string is unconditional, this whole node (the remainder of the file) can be replaced with:
```
&imx477 {
        compatible = "sony,imx378";
};
```